Sources/SwiftCurrent_SwiftUI/WorkflowItem.swift

```diff
@@ -30,32 +30,42 @@ import SwiftCurrent
  ```
  */
 @available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
-public final class WorkflowItem<F: FlowRepresentable & View> {
+public final class WorkflowItem<F: FlowRepresentable & View, Content: View> {
     var metadata: FlowRepresentableMetadata!
     private var flowPersistenceClosure: (AnyWorkflow.PassedArgs) -> FlowPersistence = { _ in .default }
     private var modifierClosure: ((AnyFlowRepresentableView) -> Void)?
 
     /// Creates a `WorkflowItem` with no arguments from a `FlowRepresentable` that is also a View.
-    public init(_: F.Type) {
+    public init(_: F.Type) where Content == F {
         metadata = FlowRepresentableMetadata(F.self,
                                              launchStyle: .new,
                                              flowPersistence: flowPersistenceClosure,
                                              flowRepresentableFactory: factory)
     }
 
+    private init(metadata: FlowRepresentableMetadata,
+                 persistence: @escaping (AnyWorkflow.PassedArgs) -> FlowPersistence,
+                 modifier: ((AnyFlowRepresentableView) -> Void)?) {
+        self.metadata = metadata
+        flowPersistenceClosure = persistence
+        modifierClosure = modifier
+    }
+
     /**
      Provides a way to apply modifiers to your `FlowRepresentable` view.
 
      ### Important: The most recently defined (or last) use of this, is the only one that applies modifiers, unlike onAbandon or onFinish.
      */
-    public func applyModifiers<V: View>(@ViewBuilder _ closure: @escaping (F) -> V) -> Self {
+    public func applyModifiers<V: View>(@ViewBuilder _ closure: @escaping (F) -> V) -> WorkflowItem<F, V> {
```

Comment:
This is no longer `Self`?  But it looks like `Self`.

Reply:
Looks like, but isn't The trick here is that `V` representing the result of the modifier block is now stored on the `WorkflowItem`. Previously the second specialization of `WorkflowItem` was almost certainly the `FlowRepresentable & View` but now it it's most likely a `ModifiedContent<..., FlowRepresentable & View>`